src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      return calculator.result + value;
+    },
```

Comment:
Make it like a pure function.
Let this method return you a pure result, and save it in your operate method.
For example
```
add(num1, num2) {
  return num1 + num2;
}
```